packages/hooks/src/useP2PAdvertList.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useFetch, useInvalidateQuery } from '@deriv/api';
+import useExchangeRate from './useExchangeRate';
+
+/**
+ * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
+ */
+const useP2PAdvertList = () => {
+    const { getRate } = useExchangeRate();
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useFetch('p2p_advert_list');
```

Comment:
@niloofar-deriv I think we should be using [usePaginatedFetch](https://github.com/binary-com/deriv-app/blob/feature/wallets_with_traders_hub_2/packages/api/src/usePaginatedFetch.ts) 🤔